pkg/api/kafka_instance_types.go

```diff
@@ -0,0 +1,22 @@
+package api
+
+type SupportedKafkaInstanceType struct {
+	Id                  string               `json:"id"`
+	SupportedKafkaSizes []SupportedKafkaSize `json:"sizes"`
+}
+
+type SupportedKafkaSize struct {
```

Comment:
Do we need this? We already have these types in the kafka supported instance types config: [KafkaInstanceType](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/larger-kafka-instances/internal/kafka/internal/config/kafka_supported_instance_types.go#L14) and [KafkaInstanceSize](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/larger-kafka-instances/internal/kafka/internal/config/kafka_supported_instance_types.go#L47)

Reply:
I don't think this is used anywhere apart from the `GetSupportedKafkaInstanceTypesByRegion()` and unit test which are both in the internal package so they shouldn't need to be visible at this level.

Reply:
I'll try using the config types. Thanks

Reply:
Ah didn't realize that those are not string values any longer. I think these can be moved into the internal package though since it's not used by other services like connectors. Maybe we can move this into the `internal/kafka/internal/api`?